dev/reloadable-server/src/main/scala/play/core/server/LagomReloadableDevServerStart.scala

```diff
@@ -83,6 +84,14 @@ object LagomReloadableDevServerStart {
           }
         }
 
+        val before = System.currentTimeMillis()
+        val address = InetAddress.getLocalHost
+        val after = System.currentTimeMillis()
+        if (after - before > 100) {
+          System.out.println(s"WARNING: Retrieving local host name ${address} took more than 100ms, this can create problems at startup with Lagom. \r\n" +
```

Comment:
Maybe we can extract the 100 magic number to a constant. That should help prevent any issues if someone updates the threshold but forgets to change the message.